requirements/dev.txt

```diff
@@ -1,4 +1,5 @@
 -r base.txt
+django-allauth==0.40.0
```

Comment:
This is already in base.txt. And even if it wasn't, this would need to go there (not into dev)